### PR TITLE
add --jwts-filename flag to use pre-existing jwts from a specific file

### PR DESCRIPTION
**Pull request by nmaltais**
Tests are failing because GET_RSA_JWT throws "jwt_generator: retrieving; corrupt jwt cache, wanted 8 bytes got 0"

I suspect this was happening before but it is only getting caught now because of this piece added in payloader.handleReqs()

image

I'm not sure why it's happening though

### CODE REVIEW

File: pkgs/jwt-generator/jwt.go [hunk 1]
```diff
@@ -32,6 +34,7 @@ type Config struct {
 	JwtCustomClaimsJSON string
 	JwtIss     			    string
 	JwtAud     			    string
+	JwtsFilename        string
```
Comment (domsolutions) [inline pkgs/jwt-generator/jwt.go:37]: NIT spacing

File: cmd/payloader/run.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ const (
 	argJWTAud       	 = "jwt-aud"
 	argJWTHeader    	 = "jwt-header"
 	argJWTKid       	 = "jwt-kid"
+	argJWTsFilename    = "jwts-filename"
```
Comment (domsolutions) [inline cmd/payloader/run.go:31]: spacing

**Review by domsolutions — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -90,6 +90,7 @@ Flags:
       --jwt-kid string           JWT KID
       --jwt-sub string           JWT subject (sub) claim
       --jwt-claims string        JWT custom claims as a JSON string, ex: {"iat": 1719410063, "browser": "chrome"}
+  -f, --jwts-filename string     File name in .cache where the JWTs to use are stored
```
[the remark this answers is not in the document]
Comment (nmaltais) [reply README.md:93]: Thanks! Had forgotten to edit the README

File: pkgs/jwt-generator/jwt.go [hunk 4]
```diff
@@ -58,6 +61,65 @@ func (c *Config) validate() error {
...
+			errs <- err
+			close(errs)
+			close(recv)
+			return recv, errs
+		}
+
+		// Parse file lines for JWTs
+		scanner := bufio.NewScanner(file)
+		// JWT Regex
+		jwtRegex, _ := regexp.Compile(`[\w-]{2,}\.[\w-]{2,}\.[\w-]{2,}`)
```
[the remark this answers is not in the document]
Comment (nmaltais) [reply pkgs/jwt-generator/jwt.go:92]: True, and the sample file in /test should be enough to show a valid file format

**Pull request merged**